src/aspire/utils/rotation.py

```diff
@@ -31,6 +33,7 @@ def __init__(self, matrices):
         # Attribute indicating if rotation matrices are about the z-axis,
         # which will cause a gimbal lock warning for ZYZ convention.
         self._about_z = False
```

Comment:
I think we can drop this flag and its use on line 299.

Reply:
sure, just pushed.